### PR TITLE
Fork backtrace-rs

### PR DESCRIPTION
**Pull request by hehechen**
Signed-off-by: hehechen <awd123456sss@gmail.com>
Closes: https://github.com/tikv/pprof-rs/issues/184
Fork [backtrace-rs](https://github.com/rust-lang/backtrace-rs) and modify [MAPPINGS_CACHE_SIZE](https://github.com/rust-lang/backtrace-rs/blob/5be2e8ba9cf6e391c5fa45219fc091b4075eb6be/src/symbolize/gimli.rs#L55) to 10.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -81,3 +81,6 @@ harness = false
 
 [package.metadata.docs.rs]
 all-features = true
+
+[patch.crates-io]
+backtrace = { git = 'https://github.com/hehechen/backtrace-rs' }
```
Comment (YangKeao) [inline Cargo.toml:86]: It's not acceptable for pprof-rs to use a git dependency. Please publish it on crates.io :thinking:

Comment (hehechen) [reply Cargo.toml]: > It's not acceptable for pprof-rs to use a git dependency. Please publish it on crates.io 🤔

Done. https://crates.io/crates/pprof-backtrace

**Review by YangKeao — APPROVED** [review]
LGTM

**Comment by YangKeao** [comment]
Wait a minute. https://github.com/tikv/pprof-rs/pull/188 will fix the CI.

**Comment by sticnarf** [comment]
I'm worrying whether there will be problems if the user links different backtrace-rs libraries.
@hehechen Is it acceptable for you to patch at your binary? And we update backtrace-rs until unstream accepts your patch.

**Comment by hehechen** [comment]
I will patch to TiKV and TiFlash proxy. https://github.com/tikv/tikv/pull/14026

**Comment by hehechen** [comment]
I will patch to TiKV and TiFlash proxy. https://github.com/tikv/tikv/pull/14026

**Pull request closed without merging**